.github/workflows/stale.yml

```diff
@@ -14,4 +14,5 @@ jobs:
           close-pr-message: 'This PR was closed because it has been open for 30 days with no activity.'
           start-date: '2023-08-20T00:00:00Z' #PRs before stale bot introduced should be ignored
           days-before-pr-close: 30
-          days-before-pr-stale: 20
\ No newline at end of file
+          days-before-pr-stale: 20
+          days-before-issue-stale: -1
```

Comment:
As an alternative, I believe you can use the `on` field at the top to limit this to only PRs.
e.g. https://github.com/DataDog/dd-trace-go/blob/main/.github/workflows/ecosystems-label.yml#L2